base/system/format/format.c

```diff
@@ -435,8 +434,8 @@ int wmain(int argc, WCHAR *argv[])
             return -1;
 
         case DRIVE_NO_ROOT_DIR:
-            K32LoadStringW(GetModuleHandle(NULL), STRING_NO_VOLUME, szMsg, ARRAYSIZE(szMsg));
-            PrintWin32Error(szMsg, GetLastError());
+            ConResPuts(StdErr, STRING_NO_VOLUME);
+            ConPuts(StdErr, L".\n");
```

Comment:
Here and in .rc files, is it correct to add a full-stop to all languages?

Reply:
The full stop must be added only in the rc

Reply:
Reverted.